convert/currency/src/convert.currency.plugin.ts

```
@@ -0,0 +1,174 @@
+  if (!config.sourceCurrency) throw new Error('Source currency is required')
+  if (!config.targetCurrency) throw new Error('Target currency is required')
+  if (!config.amountField) throw new Error('Amount field is required')
+  if (!config.convertedAmountField)
+    throw new Error('Converted amount field is required')
+}
+
+export function validateAmount(amount: any): { value: number } | { error: string } {
+  if (!amount) {
+    return { error: 'Amount is required' }
+  }
+  if (isNaN(amount)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)